lib/manageiq/cross_repo/test_core.rb

```diff
@@ -0,0 +1,29 @@
+module ManageIQ::CrossRepo
+  class TestCore < TestBase
+    attr_reader :core_repo, :plugin_repos
+
+    def initialize(core_repo, plugin_repos)
+      @core_repo = Repository.new(core_repo)
+      @plugin_repos = plugin_repos.to_a.map { |repo| Repository.new(repo) }
+    end
+
+    def run
+      ensure_repo(core_repo)
+      plugin_repos.each { |plugin_repo| ensure_repo(plugin_repo) }
+
+      File.write(core_repo.path.join("bundler.d", "overrides.rb"),
+        plugin_repos.map { |plugin| "override_gem \"#{plugin.repo}\", :path => \"#{plugin.path}\"" }.join("\n")
+      ) unless plugin_repos.empty?
```

Comment:
This feels kind of clunky...perhaps?

```ruby
      content = plugin_repos.map { |plugin| "override_gem \"#{plugin.repo}\", :path => \"#{plugin.path}\"" }.join("\n")
      File.write(core_repo.path.join("bundler.d", "overrides.rb"), content) if content.any?
```

Additionally, can we even have a case where the plugin_repos is empty?  That is, I thought we ensured that we have at least one plugin repo (maybe that check should be in the initialize here if not?)

Reply:
Yeah I'm checking in the Rakefile but I'll add a check in the initializer also

Reply:
Done in 3833683